src/Stryker.CLI/Stryker.CLI/Clients/StrykerNugetFeedClient.cs

```diff
@@ -0,0 +1,40 @@
+using Newtonsoft.Json;
+using NuGet.Versioning;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Stryker.CLI.Clients
+{
+    public interface IStrykerNugetFeedClient
+    {
+        Task<SemanticVersion> GetMaxVersion();
+    }
+
+    public class StrykerNugetFeedClient : IStrykerNugetFeedClient
+    {
+        private const string NugetStrykerFeed = "https://api.nuget.org/v3-flatcontainer/dotnet-stryker/index.json";
+
+        public class StrykerNugetFeed
+        {
+            [JsonProperty("versions")]
+            public readonly List<string> Versions = default;
+        }
+
+        public async Task<SemanticVersion> GetMaxVersion()
+        {
+            using var httpclient = new HttpClient();
+            try
+            {
+                var json = await httpclient.GetStringAsync(NugetStrykerFeed);
+                var instance = JsonConvert.DeserializeObject<StrykerNugetFeed>(json);
+                return instance.Versions.Max(v => SemanticVersion.Parse(v));
+            }
+            catch
+            {
+                return null;
```

Comment:
What do you suggest this should return if we cannot determine the latest version?

Reply:
Ah wait you're suggesting to let the caller handle the exception. Well, we explicitly did not want to burder the caller with having to deal with exceptions due to for example there not being a network connection. I would prefer actually to return MaxInt instead of null so the caller never has to worry about this nuget client to be functional.

Reply:
Well no I suggest just throwing here if it cannot reach the destination for whatever reason - and then have `try-catch` in the calling code. Alternatively, this can be renamed to `GetThisStuffOrDefault` but I don't see a need for this.

Reply:
Hah I answered too fast. Yeah MaxInt can also probably work here.

Reply:
Hmm I don't like maxint. We currently just check if the currentversion != maxversion. So if we return maxversion we will print that there is a new version available. Even though there might not be a new version. I don't see the harm in returning null

Reply:
Sorry I meant MinInt of course. If you return MinInt then the current version will always be newer and nothing will be printed. The problem with returning null is that you now force the caller to deal with null.

Reply:
In the end I just went for version 0.0.0 as default